golem_messages/message.py

```diff
@@ -83,6 +83,12 @@ def deserialize_task_failure(key, value):
     return value
 
 
+def deserialize_cannot_compute_task(key, value):
+    if key == 'cannot_compute_task':

```

Comment:
will these string values only be used in this file?

If not it might be good to make it a constant or enum.

Reply:
They are defined in `__slots__` (key is an attribute name)